include/sonar_driver/sonardevices/OculusM1200d.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <sonar_driver/sonardevices/OculusSonar.h>
+
+class OculusM1200d : public OculusSonar
+{
+public:
+    OculusM1200d();
```

Comment:
@EulersBoiler Is this file still used anywhere or does it have an actual implementation? If not, I think we can remove it.